webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/reports/StatusReports.scala

```diff
@@ -342,6 +342,7 @@ final case class BlockStatusReport (
 }
 final case class ValueStatusReport  (
     componentName      : String
+  , unexpanded         : String
```

Comment:
i'd rather have unexpandedComponentName, otherwise we may confuse it with the unexpandedValue

Reply:
I'll go with "ExpectedComponentName" with is more clear to me the unexpanded. Same for values